CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 - Updates the background color for the `"iconOnly"` and `"text"` variants of the `Button` component.
 - Updates the DOM in the header of the `FeedbackBox` component to improve accessibility.
 - Updates the `Link` component to include descriptive text for screen readers in the component's `"external"` variant.
+- Truncate breadcrumb text if beyond 60 characters then add ellipsis at the end.
+  If the breadcrumb text is truncated, the DS `Tooltip` component is used to display the full text in a tooltip when the text is hovered.
```

Comment:
Nitpick: instead of creating a new line for this sentence, this can just go at the end of the previous line.

Reply:
Ok